modules/touch/src/tuioear.cpp

```diff
@@ -42,67 +44,29 @@ void TuioEar::removeTuioObject(TuioObject*) { }
 
 void TuioEar::addTuioCursor(TuioCursor* tcur) {
     _mx.lock();
-    _tap = false;
-    // find same id in _list if it exists in _removeList (new input with same ID as a
-    // previously stored)
-    long i = tcur->getSessionID();
-    std::vector<long>::iterator foundID = std::find_if(
-        _removeList.begin(),
-        _removeList.end(),
-        [&i](long id) { return id == i; });
-
-    // if found, remove id from _removeList and update, otherwise add new id to list
-    if (foundID != _removeList.end()) {
-        std::find_if(
-            _list.begin(),
-            _list.end(),
-            [&i](const TuioCursor& cursor) {
-                return cursor.getSessionID() == i;
-            }
-        )->update(tcur);
-        _removeList.erase(foundID);
-    }
-    else {
-        _list.emplace_back(*tcur);
-    }
+    TouchInput input(size_t(tcur->getTuioSourceID()),
```

Comment:
This snippet occurs in 3 different places, maybe move it to its own function

Reply:
Resolved in e33004e